blocks/text.js

```diff
@@ -647,6 +664,80 @@ Blockly.Constants.Text.TEXT_QUOTES_EXTENSION = function() {
   this.quoteField_('TEXT');
 };
 
+/**
+ *
+ * @mixin
+ * @package
+ * @readonly
+ */
+Blockly.Constants.Text.PILCROW_IMAGE_MIXIN = {
```

Comment:
Instead of having this as a mixin I think it's better to just define an image field on the block. This will make the src path static, but I'm planning on adding dynamic media path support to the JSON at some point, so it will only be temporary.